### PR TITLE
[1804] Fill up all name_en fields on pep related companies

### PR DESCRIPTION
**Pull request by Andrii17s**

### CODE REVIEW

**Review by criticus — APPROVED** [review]

File: business_register/converter/pep.py [hunk 1]
```diff
@@ -578,6 +581,8 @@ def save_or_update_peps_companies(self, peps_companies_data):
                     company.antac_id = company_antac_id
                     company_update_fields.append('antac_id')
             if not company:
+                if company_name_en is None:
+                    company_name_en = transliterate(company_name)
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:585]: first what you need: pass comany_name to `translate_company_type_in_string`

File: business_register/converter/pep.py [hunk 2]
```diff
@@ -578,6 +581,8 @@ def save_or_update_peps_companies(self, peps_companies_data):
                     company.antac_id = company_antac_id
                     company_update_fields.append('antac_id')
             if not company:
+                if company_name_en is None:
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:584]: `company_name_en` can be empty string for example

File: business_register/converter/pep.py [hunk 3]
```diff
@@ -589,6 +594,9 @@ def save_or_update_peps_companies(self, peps_companies_data):
                 if company.name_en != company_name_en:
                     company.name_en = company_name_en
                     company_update_fields.append('name_en')
+                if company.name_en is None:
+                    company.name_en = transliterate(company_name)
+                    company_update_fields.append('name_en')
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:599]: same

File: business_register/converter/pep.py [hunk 4]
```diff
@@ -15,6 +15,9 @@
 from data_ocean.utils import to_lower_string_if_exists
 from location_register.converter.address import AddressConverter
 from stats.tasks import endpoints_cache_warm_up
+from data_ocean.transliteration.utils import transliterate, translate_company_type_in_string,\
+    translate_country_in_string
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:19]: unused imports

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/converter/pep.py [hunk 5]
```diff
@@ -578,6 +580,8 @@ def save_or_update_peps_companies(self, peps_companies_data):
                     company.antac_id = company_antac_id
                     company_update_fields.append('antac_id')
             if not company:
+                if company_name_en is None and company_name:
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:583]: what if company company_name_en is empty string?
```suggestion
                if not company_name_en:
```

**Review by Iva-khar — APPROVED** [review]

File: business_register/converter/pep.py [hunk 6]
```diff
@@ -589,6 +593,9 @@ def save_or_update_peps_companies(self, peps_companies_data):
                 if company.name_en != company_name_en:
                     company.name_en = company_name_en
                     company_update_fields.append('name_en')
+                if not company.name_en and company_name:
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:596]: here?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/converter/pep.py [hunk 7]
```diff
@@ -564,6 +566,9 @@ def save_or_update_peps_companies(self, peps_companies_data):
             country_name = link[10]
             source_id = link[11]
             company_name_en = link[12]
+            if not company_name_en:
+                company_name_en = transliterate(translate_company_type_in_string(company_name))
+                company_update_fields.append('name_en')
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:571]: why?
`company_update_fields.append('name_en')`

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**